api/api.ts

```diff
@@ -197,4 +282,8 @@ function getFilename(req: express.Request): string {
     }
 }
 
-export default app
+function isIPInvalid(clientIP: string): boolean {
+    return clientIP !== whitelistedIP
```

Comment:
I guess checking if an ip is invalid will be a little more than just a simple string comparison. For example, for `localhost` , the value of clientIP will be `::1` , a value completely unexpected to me

Reply:
It will work if we fill .env like this way: `::ffff:<number_address>`